static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -86,16 +93,30 @@ define([
         }, {});
     }
 
+    function summariseAbTests(testsAndVariants) {
+        var testAndVariantPairs = pairs(testsAndVariants);
+        if (testAndVariantPairs.length === 0) {
+            return 'No tests running';
+        } else {
+            return map(testAndVariantPairs, function (pair) {
+                return pair[0] + '=' + pair[1];
```

Comment:
Yeah, maybe. At the time I felt the getParticipations interface was a little awkward to work with, but I guess the tuple is opaque. We could always just extract the keys and use them like thus:

    var tests = keys(participations);
    if (tests.length === 0) {
        return 'No tests running';
    } else {
        return map(tests, function (testKey) {
            var test = participations[testKey];
            return testKey + '=' + test.variant;
        }).join(', ');
    }